blt/src/Blt/Plugin/Commands/HsCommands.php

```diff
@@ -8,13 +8,73 @@
 use Drupal\Core\Serialization\Yaml;
 use GuzzleHttp\Client;
 use Robo\Exception\TaskException;
+use Sws\BltSws\Blt\Plugin\Commands\SwsCommandTrait;
 
 /**
  * Various BLT commands for H&S stack.
  */
 class HsCommands extends BltTasks {
 
-  use HsCommandTrait;
+  use SwsCommandTrait;
+
+  /**
+   * After code deployed, update all sites on the stack.
+   *
+   * @command humsci:post-code-deploy
+   *
+   * @aliases humsci:post-code-update
+   */
+  public function postCodeDeployUpdate() {
+    $sites = $this->getConfigValue('multisites');
+    $parallel_executions = 10;
+    $site_chunks = array_chunk($sites, ceil(count($sites) / $parallel_executions));
+    $commands = [];
+    foreach ($site_chunks as $sites) {
+      $commands[] = $this->blt()
+        ->arg('humsci:update-sites')
+        ->arg(implode(',', $sites))
+        ->getCommand();
+    }
+    file_put_contents(sys_get_temp_dir() . '/update-report.txt', '');
+    $this->taskExec(implode(" &\n", $commands) . PHP_EOL . 'wait')->run();
```

Comment:
Would this need two ampersands? `&&`

Reply:
1 `&` allows parallel execution. 2 `&&` will put them into sequence, with the later only executing if the preceeding commands succeed.

Reply:
TIL. Neat 👍